.travis.yml

```diff
@@ -1,5 +1,21 @@
 language: node_js
 node_js:
     - "8"
-script: ./build/ci.sh
-cache: yarn
+cache:
+  yarn: true
+  directories:
+    - node_modules
+jobs:
+  include:
+    - # commit lint
+      script: commitlint-travis
+    - # code lint
+      script: npm-run-all clean build:i18n lint
```

Comment:
How long does `build:i18n` take? Is it possible to cache that directory?

Reply:
Pretty fast, less than 5s